brands/amenity/fast_food.json

```diff
@@ -1228,6 +1228,23 @@
       "official_name": "Papa Murphy's Take 'N' Bake Pizza",
       "takeaway": "yes"
     }
+  },
+    "amenity/fast_food|Penn Station": {
+    "nocount": true,
+    "countryCodes": ["us"],
+    "match": [
+      "amenity/fast_food|Penn Station",
```

Comment:
This can be removed because this is the actual name of the entry...

Reply:
Oh... I'm sorry. I meant that only `amenity/fast_food|Penn Station` should be removed.
`amenity/restaurant|Penn Station` should stay there 😄